### PR TITLE
Repository documentation

### PR DESCRIPTION
**Pull request by jzigmund**
Added docstrings to Repository related files

### CODE REVIEW

**Comment by jzigmund** [comment]
@vinzenz contains a few missing doc parts (marked with `?`), tomorrow we can walk through the PR and revise those missing ones and fix others (as with previous PR) :mag:

File: leapp/repository/loader.py [hunk 1]
```diff
@@ -3,6 +3,17 @@
 
 
 def library_loader(mod, prefix, paths):
+    """
+    Prepares modules and packages to be loaded
+
+    :param mod: module namespace
+    :type mod: ?
```
Comment (vinzenz) [inline leapp/repository/loader.py]: Python Module

File: leapp/repository/loader.py [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
+    :param prefix: prefix of module/package
+    :type prefix: str
+    :param paths: iterable paths
+    :type paths: map object?
```
Comment (vinzenz) [inline leapp/repository/loader.py]: tuple(str)

File: leapp/repository/manager.py [hunk 3]
```diff
@@ -1,56 +1,100 @@
...
+
+        :param repo: Repository that will be added (registered) 
+        :type repo: :py:class:`leapp.repository.Repository`
+        """
         self._repos.append(repo)
 
     @property
     def repos(self):
+        """
+        :note: not implemented yet ?
```
Comment (vinzenz) [inline leapp/repository/manager.py]: Still TODO

File: leapp/repository/manager.py [hunk 4]
```diff
@@ -1,56 +1,100 @@
...
 
     def dump(self):
+        """
+        :return: List of resources in all known repositories
+        """
         return [repo.dump() for repo in self._repos]
 
     def inject(self):
+        """
+        :note: repo.inject() is not implemented ?
```
Comment (vinzenz) [inline leapp/repository/manager.py]: TODO

**Pull request merged**